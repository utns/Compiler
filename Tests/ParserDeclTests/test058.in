procedure foo(a, b, c: float);
begin
end;