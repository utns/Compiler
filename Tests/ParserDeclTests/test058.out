integer                  type 
float                    type 
char                     type 
foo                       proc
args:          
     b                       param           float
     c                       param           float
     a                       param           float
locals:        
